### ISSUE TITLE
Search in the Docs of Pycord

### ISSUE THREAD

**Issue by leon-3** [PyDrocsid/cogs#160]
**Is your feature request related to a problem? Please describe.**
I think, more people will switch to a discord.py fork like [pycord](https://github.com/Pycord-Development/pycord), therefore it would be usefull to have a `.pycord_docs` command to search in the pycord docs just like in the discord.py docs. 

**Describe the solution you'd like**
As described above a `.pycord_docs` command (aliases: `pycord`, `pyc`) just like the `.dpy_docs` command.

**Additional context**
From "Lynix152#9707":

ich glaub das ist nicht viel, das ist nur in line 183 was dazunehmen: 
```py
async def do_rtfm(ctx: Context, key: str, obj: Optional[str]):
    page_types = {"discord.py": "https://discordpy.readthedocs.io/en/latest", "python": "https://docs.python.org/3 ", "pycord": "https://pycord.readthedocs.io/en/latest/index.html"}
```
 (Die doc von PyCord und dpy ist ja fast gleich aufgebaut)

```py
@commands.command(aliases=["pyc"])
    async def pyc(self, ctx: Context, *, obj: str = None):
        """
        search the official python documentation
        """

        await do_rtfm(ctx, "pycord", obj)
```

---
Maybe a rtfm command for other discord.py forks like [nextcord](https://github.com/nextcord/nextcord) could also be usefull.

**Comment by NekoFanatic** [comment]
I would do it

**Comment by Defelo** [comment]
fixed in da787ce
